app/questionnaire/value_source_resolver.py

```diff
@@ -104,7 +104,7 @@ def _resolve_answer_value_source(
         answer_id = value_source["identifier"]
 
         answer_value = self._get_answer_value(
-            answer_id=answer_id, list_item_id=list_item_id
+            answer_id=answer_id, list_item_id=list_item_id, assess_routing_path=True
```

Comment:
This can't now just be True, should only be True for the cases it needs the path, i.e for placeholder, just the placeholders we care about.

Reply:
Updated should only be true when its a placeholder transform we want

Reply:
Resolving on behalf of Mebin.